src/Raven.Server/Documents/DocumentDatabase.cs

```diff
@@ -1576,9 +1576,20 @@ private void LoadTimeSeriesPolicyRunnerConfigurations()
             IDatabaseTask configuration,
             IDatabaseTaskStatus taskStatus,
             bool keepTaskOnOriginalMemberNode = false)
+        {
+            return WhoseTaskIsIt(ServerStore, databaseTopology, configuration, taskStatus, NotificationCenter, keepTaskOnOriginalMemberNode);
+        }
+
+        public static string WhoseTaskIsIt(
```

Comment:
Why do we need a new overload?
Can we move this to a helper class? And everyone will call it directly.

Reply:
Done.